lua/wikis/ageofempires/Infobox/League/Custom.lua

```diff
@@ -117,16 +117,18 @@ function CustomInjector:parse(id, widgets)
 	return widgets
 end
 
----@return string?
+---@return Html?
 function CustomLeague:createBottomContent()
 	local yesterday = os.date('%Y-%m-%d', os.time() - SECONDS_PER_DAY)
 
 	if self.data.endDate and yesterday <= self.data.endDate then
-		return MatchTicker.get{args={
-			parent = self.pagename,
+		---@diagnostic disable-next-line: missing-fields
```

Comment:
which fields are missing and why?

Reply:
The match ticker expects a frame

Reply:
Hmm, if i change the param annotation to `table|Frame`, my LuaLS still complains here without the diag disabled

Reply:
if you change the anno of `CustomMatchTicker.tournament` to allow `table|Frame` and use something like the following it should work:
```lua
		local tickerArgs = {
			tournament = self.pagename,
			limit = tonumber(self.args.matchtickerlimit) or 7,
			infoboxWrapperClass = 'false',
			infoboxClass = true
		}
		return MatchTicker.tournament(tickerArgs)
```
that way it doesn't try to make it a frame, but accepts it is a plain table ...

Reply:
Apparently it wouldn't even need the `table` annotation, it just happily takes the table while expecting a frame, if passed as a variable